doc/Apptools_Test_Suite_Developer_Guide.md

```diff
@@ -0,0 +1,68 @@
+# Apptools Test Suite Developer Guide
+
+## Overview
+
+This document is intended for developers who contribute Apptools test cases development.
```

Comment:
This document is intended for those who want to make test contributions to App-Tools test suite.